<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseTheme</class>
 <widget class="QWidget" name="ChooseTheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="themeHP">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string>快乐小狗主题</string>
   </property>
  </widget>
  <widget class="QLabel" name="laPKU">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="laHP">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>221</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PKU">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>北京大学主题</string>
   </property>
  </widget>
  <widget class="QLabel" name="laHPDOG">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HappyDog">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>哈利波特主题</string>
   </property>
  </widget>
  <widget class="QLabel" name="laQIDAI">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
